ipn/ipnlocal/peerapi.go

```diff
@@ -309,6 +310,10 @@ func (h *peerAPIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.handlePeerPut(w, r)
 		return
 	}
+	if strings.HasPrefix(r.URL.Path, "/v0/partial-files/") {
```

Comment:
Nit: perhaps move this above `/v0/put` since it's both chronologically called first and also the declaration below comes first?